minetest/minetest
# TouchScreenGUI: Add an exit / "ESC" button to the "rare controls" bar

Currently, Android users have to press the Android back button to open the pause menu. This works, but new users often\* don't discover it by themselves and instead close and restart the app when they want to go back to the main menu.

This PR adds a second way of opening the pause menu: A button in the so-called "rare controls" bar.

![screenshot: rare controls bar 1](https://github.com/minetest/minetest/assets/82708541/53a9ebd9-5fd9-46dc-83dc-f6f2f931802f)

![screenshot: rare controls bar 2](https://github.com/minetest/minetest/assets/82708541/b6283eaf-e00f-4480-b067-5cb12422f11f)

Fixes #12070, but this PR doesn't cover all the points mentioned in that issue.

\* I don't know how often, of course, but I had this problem myself.

## To do

This PR is a Ready for Review.

## How to test

Open the "rare controls" bar and press the exit button. Verify that it opens the pause menu.
